linter.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+from typing import Pattern, List
```

Comment:
Thanks for including type hints! We should probably include mypy in our CI checks. Can you open an issue for that so we don't forget?

Reply:
Yep, I created https://github.com/oppia/oppia-web-developer-docs/issues/97 for this. PTAL